src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -509,15 +507,6 @@ public void setCryptograph(@Nullable CryptographFactory cryptograph) {
     this.cryptograph = cryptograph;
   }
 
-  public int getMgmtApiMetricsPort() {
-    return mgmtApiMetricsPort == null ? DEFAULT_MGMT_API_METRICS_PORT : mgmtApiMetricsPort;
-  }
-
-  @JsonProperty("mgmtApiMetricsPort")
```

Comment:
I presume this is going because we can't configure the port at the moment?

Reply:
No, I moved it under the HttpManagement class where rest of the mgmtApi related settings are

Reply:
Sorry, forgot to delete this comment after seeing where you'd moved it, all good.